howto/install-appliance/aws.md

```diff
@@ -0,0 +1,136 @@
+You can install the Anbox Cloud Appliance on AWS in one of two ways:
+
+- Install through the AWS Marketplace. This is the recommended way, because this method simplifies the installation and deployment process and allows billing to be handled directly through AWS.
+- Install the Anbox Cloud Appliance snap on an AWS machine. This method is not recommended, but if you want to do it anyway, see the [Install the Anbox Cloud Appliance on your local machine](https://discourse.ubuntu.com/t/install-appliance/22681) tutorial for instructions on how to install the snap.
+
+The following instructions guide you through all relevant steps to deploy the Anbox Cloud Appliance from the AWS Marketplace. For additional information, see the [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/launching-instance.html) about launching an instance.
+
+The entire deployment process will take 10-15 minutes, depending on the selected hardware and the network conditions.
+
+## Before you start
+
+Deploying the Anbox Cloud Appliance requires some familiarity with AWS. In particular, you should be familiar with:
+
+- Amazon Elastic Compute Cloud (Amazon EC2), for basic EC2 configuration
+- Amazon Elastic Block Storage (Amazon EBS), for configuring the EC2 instance storage and managing encryption
+- Amazon Virtual Private Cloud (Amazon VPC), for configuring an internet facing subnet and a security group
+
+The appliance uses the following billable services by AWS:
+
+- EC2 and Marketplace appliance (see the AWS Marketplace product page for costs)
+- Network egress
+
+You do not need to request an increase in limits for your AWS account for this deployment.
```

Comment:
We have seen some customers not having access to some of the instance types (e.g. `g4dn`) and had to file a support ticket to get access.

Reply:
When somebody just signs up a new account with AWS his quota will be limited and he wont be able to launch all instance types. Before he can do so he needs to ask AWS support to get his quota bumped.

But fine to drop this here as it goes into AWS specifics we don't have to cover here.